boards/mikroe/quail/Kconfig.defconfig

```diff
@@ -0,0 +1,14 @@
+# MikroE Quail board configuration
```

Comment:
change commit title to something descriptive like `boards: mikro: Add quail board`